testsuite/features/secondary/proxy_container_cobbler_pxeboot.feature

```diff
@@ -0,0 +1,139 @@
+    When I start local monitoring of Cobbler
+
+  # We currently test Cobbler with SLES 15 SP4, even on Uyuni
+  Scenario: Install the TFTP boot package on the server for Cobbler tests
+    When I install package tftpboot-installation on the server
+    And I wait for "tftpboot-installation-SLE-15-SP4-x86_64" to be installed on "server"
+
+  Scenario: Create auto installation distribution
+    When I follow the left menu "Systems > Autoinstallation > Distributions"
+    And I follow "Create Distribution"
+    And I enter "SLE-15-SP4-TFTP" as "label"
+    And I enter "/usr/share/tftpboot-installation/SLE-15-SP4-x86_64/" as "basepath"
+    And I select "SLE-Product-SLES15-SP4-Pool for x86_64" from "channelid"
+    And I select "SUSE Linux Enterprise 15" from "installtype"
+    And I click on "Create Autoinstallable Distribution"
+    Then I should see a "Autoinstallable Distributions" text
+    And I should see a "SLE-15-SP4-TFTP" link
+
+  # WORKAROUND bsc#1195842
+  # Default cobbler kernel parameters are wrong in case of proxy
+  Scenario: Fix kernel parameters
+    When I follow the left menu "Systems > Autoinstallation > Distributions"
+    And I follow "SLE-15-SP4-TFTP"
+    And I enter "useonlinerepo insecure=1 install=http://proxy.example.org/ks/dist/SLE-15-SP4-TFTP self_update=http://proxy.example.org/ks/dist/child/sle15-sp4-installer-updates-x86_64/SLE-15-SP4-TFTP" as "kernelopts"
+    And I click on "Update Autoinstallable Distribution"
+    Then I should see a "Autoinstallable Distribution Updated" text
+
+  Scenario: Create auto installation profile
+    When I follow the left menu "Systems > Autoinstallation > Profiles"
+    And I follow "Upload Kickstart/AutoYaST File"
+    And I enter "15-sp4-cobbler" as "kickstartLabel"
+    And I select "SLE-15-SP4-TFTP" from "kstreeId"
+    And I attach the file "/sle-15-sp4-autoyast.xml" to "fileUpload"
+    And I click on "Create"
+    Then I should see a "Autoinstallation: 15-sp4-cobbler" text
+    And I should see a "Autoinstallation Details" text
+
+  Scenario: Configure auto installation profile
+    When I enter "self_update=0" as "kernel_options"
+    And I click on "Update"
+    And I follow "Variables"
+    And I enter "distrotree=SLE-15-SP4-TFTP\nregistration_key=1-TERMINAL-KEY-x86_64\nredhat_management_server=proxy.example.org" as "variables" text area
+    And I click on "Update Variables"
+    And I follow "Autoinstallation File"
+    Then I should see a "SLE-15-SP4-TFTP" text
+
+  Scenario: Migration of cobbler settings
+    Given cobblerd is running
+    And cobbler settings are successfully migrated
+    When I restart cobbler on the server
+    Then service "cobblerd" is active on "server"
+
+  Scenario: Set up tftp installation and synchronize it
+    When I configure tftp on the "server"
+    And I run Cobbler sync with error checking
+
+  Scenario: PXE boot the PXE boot minion
+    Given I set the default PXE menu entry to the target profile on the "server"
```

Comment:
When

Reply:
Thank you! As usual we probably have same issue on the "traditional" version...

Reply:
Done